dltnchecker/harvest.py

```diff
@@ -134,7 +134,44 @@ def __write_bad_records_to_log(bad_record):
         return
 
 
-class DCTester:
+class MetadataTester:
+    def __init__(self, document):
+        self.document = document
+
+    @staticmethod
+    def check_standard_rights(rights_statement):
+        rights_statements = ('http://rightsstatements.org/vocab/InC/1.0/',
+                             'http://rightsstatements.org/vocab/InC-OW-EU/1.0/',
+                             'http://rightsstatements.org/vocab/InC-EDU/1.0/',
+                             'http://rightsstatements.org/vocab/InC-NC/1.0/',
+                             'http://rightsstatements.org/vocab/InC-RUU/1.0/',
+                             'http://rightsstatements.org/vocab/NoC-CR/1.0/',
+                             'http://rightsstatements.org/vocab/NoC-NC/1.0/',
+                             'http://rightsstatements.org/vocab/NoC-OKLR/1.0/',
+                             'http://rightsstatements.org/vocab/NoC-US/1.0/',
+                             'http://rightsstatements.org/vocab/CNE/1.0/',
+                             'http://rightsstatements.org/vocab/UND/1.0/',
+                             'http://rightsstatements.org/vocab/NKC/1.0/',
+                             'https://creativecommons.org/licenses/by/4.0/',
+                             'https://creativecommons.org/licenses/by-sa/4.0/',
+                             'https://creativecommons.org/licenses/by-nc/4.0/',
+                             'https://creativecommons.org/licenses/by-nc-sa/4.0/',
+                             'https://creativecommons.org/licenses/by-nd/4.0/',
+                             'https://creativecommons.org/licenses/by-nc-nd/4.0/',
+                             'https://creativecommons.org/licenses/by/3.0/',
+                             'https://creativecommons.org/licenses/by-sa/3.0/',
+                             'https://creativecommons.org/licenses/by-nc/3.0/',
+                             'https://creativecommons.org/licenses/by-nc-sa/3.0/',
+                             'https://creativecommons.org/licenses/by-nd/3.0/',
+                             'https://creativecommons.org/licenses/by-nc-nd/3.0/',
+                             'https://creativecommons.org/publicdomain/zero/1.0/')
```

Comment:
In addition to CC0, there's also a public domain mark - https://creativecommons.org/publicdomain/mark/1.0/

Reply:
Thanks!